google/ggrc-core
# QUICK-FIX Update license headers

# Issue description

We need to keep our license headers up to date. This PR just makes sure we can't forget to update them in the future and updates the entire development branch. 

# Steps to test the changes

go through the repo and/or diff and make sure that all licnese headers have been updated and make sure that no code has been changed by accident.

# Solution description

I ran the command:
```bash
grep -rl "Copyright (C) 2018 Google Inc." | xargs sed -i 's/Copyright (C) 2018 Google Inc./Copyright (C) 2019 Google Inc./g'
```
on latest dev branch

# Sanity checklist




# PR Review checklist
